get_github_coderabbit_review/coderabbit_fetcher/__init__.py

```
@@ -0,0 +1,28 @@
+    "CodeRabbitFetcherError",
+    "AnalyzedComments",
+    "SummaryComment", 
+    "ReviewComment",
+]
+
+# Lazily expose CLI entry to avoid import-time side effects.
+def __getattr__(name: str):
+    if name == "main":
+        from .cli import main as _main
+        return _main
+    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)